Report Exporter/Sdl.Community.ReportExporter/Helpers/Help.cs

```diff
@@ -43,76 +41,125 @@ public static string GetStudioProjectsPath()
 				projectsPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments),
```

Comment:
Can we make this to by dynamic? In the next release of Studio this will not work.

Reply:
I've created a mecahnism to avoid hardcoding the studio version